aptos-move/aptos-vm/src/sharded_block_executor/counters.rs

```diff
@@ -21,3 +24,31 @@ pub static SHARDED_BLOCK_EXECUTION_SECONDS: Lazy<HistogramVec> = Lazy::new(|| {
     )
     .unwrap()
 });
+
+/// Count of the committed transactions since last restart.
+pub static SHARDED_BLOCK_EXECUTOR_TXN_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
Why not a histogramVec? (assuming you want stats per block)

Reply:
I was not thinking of getting a distribution per block, but sure why not.